.changeset/fuzzy-apes-talk.md

```diff
@@ -0,0 +1,5 @@
+---
+'fingerprint-pro-server-api-openapi': minor
```

Comment:
@ilfa Do I assume correctly that adding new signals is a `minor` version bump?

Reply:
I think `minor` makes sense for new features. `patch` is mostly for bug fixes.